spec/compiler/interpreter/openssl_spec.cr

```diff
@@ -0,0 +1,14 @@
+{% skip_file if flag?(:without_interpreter) %}
+require "./spec_helper"
+require "../loader/spec_helper"
+
+describe Crystal::Repl::Interpreter do
+  context "openssl" do
+    it "can require" do
+      interpret(<<-CR, prelude: "prelude")
+        require "openssl"
+        OpenSSL::SSL::Context::Server.new
+        CR
+    end
+  end
+end
```

Comment:
This spec tests the changed behaviour only coincidentally. It depends on the bindings and link flags for `LibSSL`.
It would be better to have a dedicated test specifically for this command expansion in `interpreter/lib_spec.cr`.
Let me know if you need help with that.

Reply:
Ah ok. I had the test originally in lib_spec, but thought it was a little different. I'll add it back.
Can you elaborate a little bit on how I can test the command is expanding? I'm just wondering if its a simple as checking if `echo ...` worked, or you had something else in mind?

Reply:
Yes, I think a simple example of a lib type with bindings specified via shell expansion in `ldflags` should be fine. If that links / executes, we can be sure that it works.
We're already using a custom `libsum` in `lib_spec` for testing variadic arguments. We can probably just derive from one of the existing samples, replacing `-lsum` with ``"-l`echo sum`"`` (for example).

Optionally adding trivial function to the lib because we don't need variadics here; but it's probably fine to just use an existing symbol if you like.